test-network-function/lifecycle/suite.go

```diff
@@ -375,6 +375,17 @@ func getDeployments(namespace string) (deployments dp.DeploymentMap, notReadyDep
 	return deployments, notReadyDeployments
 }
 
+//nolint:deadcode // to be used in Javier's change
+func collectNodeAndPendingPodInfo(ns string) {
+	context := common.GetContext()
+	nodeStatus, _ := utils.ExecuteCommand("oc get nodes -o json | jq '.items[]|{name:.metadata.name, taints:.spec.taints}'", common.DefaultTimeout, context)
+	common.TcClaimLogPrintf("Namespace: %s\nNode status:\n%s", ns, nodeStatus)
+	podStatus, _ := utils.ExecuteCommand("oc get pods -n "+ns+" --field-selector status.phase!=Running -o json | jq '.items[]|{name:.metadata.name, status:.status}'", common.DefaultTimeout, context)
```

Comment:
Also exclude phase=Succeeded from the results

Reply:
Right